test/builtin/arithmetic/test_basic.py

```diff
@@ -197,7 +197,7 @@ def test_directed_infinity_precedence(str_expr, str_expected, msg):
         ("I^(2/3)", "(-1) ^ (1 / 3)", None),
         # In WMA, the next test would return ``-(-I)^(2/3)``
         # which is less compact and elegant...
-        #        ("(-I)^(2/3)", "(-1) ^ (-1 / 3)", None),
+        ("(-I)^(2/3)", "(-1) ^ (-1 / 3)", None),
```

Comment:
Hmm. master returns -1 right now, so what we have is wrong. 

Do you know what part of the code fixes this?

Reply:
I guess this is related to the new rules you mention before...

Reply:
Nop. The rule at the end seems to be useless. The computation is done by `mathics.eval.arithmetic.eval_Power_number`, which now is called from `Power.eval_check` ( `mathics.builtin.arithfns.basic`,  line 591).